src/ucp/wireup/wireup.c

```diff
@@ -1118,6 +1118,10 @@ ucs_status_t ucp_wireup_init_lanes(ucp_ep_h ep, unsigned ep_init_flags,
         ep->flags |= UCP_EP_FLAG_LOCAL_CONNECTED;
     }
 
+    if (key.err_mode != UCP_ERR_HANDLING_MODE_NONE) {
```

Comment:
`if (key.err_mode != UCP_ERR_HANDLING_MODE_NONE) {` should be done inside ucp_worker_keepalive_add_ep

Reply:
ok, moved into ucp_worker_keepalive_add_ep